stories/Button.js

```diff
@@ -23,6 +23,7 @@ const buttonFlavors = [
   'success',
   'text-destructive',
   'stretch',
+  'reset',
```

Comment:
Not sure if we should add this since it is technically not a flavor. From what I understand about this, using `className` seems more approriate as it again is not a part of the "public" component API